Makefile

```diff
@@ -10,18 +10,27 @@ PLATFORM?=linux/$(ARCH)
 IMAGE_SIZE?=20G
 PACKER_TARGET?=qemu.elemental-${ARCH}
 REPO?=local/elemental-$(FLAVOR)
+BIN_REPO?=local/elemental-bin
```

Comment:
I'm not sure I see the use case for a new repo, where should this be published and why not use the elemental-toolkit image?

Reply:
Removed!